.github/auto-merge.yml

```diff
@@ -0,0 +1,83 @@
+- match:
+    dependency_name: FluentValidation
+    update_type: all
+
+- match:
+    dependency_name: Fody
+    update_type: all
+
+- match:
+    dependency_name: GitVersionTask
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.AspNetCore.Server.Kestrel
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.AspNetCore.StaticFiles
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.CodeAnalysis.CSharp.Workspaces
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.NET.Test.Sdk
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.PowerShell.5.ReferenceAssemblies
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.SourceLink.GitHub
+    update_type: all
+
+- match:
+    dependency_name: Moq
+    update_type: all
+
+- match:
+    dependency_name: NETStandard.Library
+    update_type: all
+
+- match:
+    dependency_name: NUnit
```

Comment:
I guess we can probably automerge this one, but we have to bear in mind that it is a public dependency of some of our testing packages. E.g. https://www.nuget.org/packages/NServiceBus.AcceptanceTesting/

Reply:
Is it possible to overwrite the repostandards files in a specific repo? If yes, we can exclude it from that specific repo by using a custom config

Reply:
@WilliamBZA yes it is: https://github.com/Particular/NServiceBus.Serializers.CompatTests/pull/32/files
So in a given repo we can specify that a config file will not be pulled in

Reply:
The problem with that is that you exclude the _whole_ file, so any other updates to the file will have to be manually propagated to any repo which is excluding that file.

Reply:
yep. The whole question that William asks is a bit odd. What I mean by that: dependabot.yml is a file that lives in a repo, so every repository defines a list of dependencies that are allowed and forbidden from PR's being raised. However, as we push this file using RepoStandards we lead ourselves into having one global list of dependencies.

So far if we will have a dependency that could be auto-merged in some repos and not in the others I would vote to simply leave it out of auto-merging. The list that we have, should already make the dependabot more manageable.

Reply:
> dependabot.yml is a file that lives in a repo, so every repository defines a list of dependencies that are allowed and forbidden from PR's being raised

`dependabot.yml` is also [defined in this repo](https://github.com/Particular/RepoStandards/blob/master/.github/dependabot.yml), so every repo has exactly the same file. It does not have an inclusion or exclusion list of dependencies. Dependabot raises for PR's for any dependency it is able to do so for.

> So far if we will have a dependency that could be auto-merged in some repos and not in the others I would vote to simply leave it out of auto-merging. The list that we have, should already make the dependabot more manageable.

I agree.